app/page.tsx

```diff
@@ -6,16 +6,8 @@ import Image from 'next/image'
 
 export default async function Home() {
   return <>
-    <Header/>
     <main>
-      {/* TODO: Pretty banner. I have no idea how to do this effectively. This 
```

Comment:
Definitely going to need to revisit this, and the design in general, at some point.